### PR TITLE
Insertion Sort

### PR DESCRIPTION
**Pull request by ghost**
New add InsertionSort

### CODE REVIEW

File: InsertionSort.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+public class MyInsertionSort {
```
Comment (cegerard) [inline InsertionSort.java]: This class must implement the ISortingAlgorithm interface.
Like it was done with default algorithm.

File: InsertionSort.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ 
+    public static void main(String a[]){
```
Comment (cegerard) [inline InsertionSort.java]: You can get rid of this method.
We use the main method provide in the App class of this project as a starting point.

**Review by cegerard — CHANGES_REQUESTED** [review]
You must add the algorithm package at the top of this file.

`package sortBench.algorithm`

**Review by cegerard — CHANGES_REQUESTED** [review]
You must move this file into the right directory.

src/main/java/sortBench/algorithms/insertion

**Comment by cegerard** [comment]
You have write access on this repository so you can clone directly.
It is not needed to fork it.

**Comment by ghost** [comment]
I added package, moved file into algorithm directory and I deleted useless main method.
It's better for sure!
But we didn't start implements in class of JAVA, so I don't really understand everything to implement

**Comment by cegerard** [comment]
@Sylvain-Reig ok it is not problem I do it on purpose to show you how a code review is done and to force you to iterate over your code. The goal here is not to implement all issue correctly but play with pull request.

**Comment by cegerard** [comment]
@Sylvain-Reig you certainly forgot to add something because your PR does not contain any modification.

Look at the `file changed` tab there nothing...

**Comment by cegerard** [comment]
@Sylvain-Reig another advise. In Java a class must have the same name than its file.
If you want that automated tests valided your PR you have to rename your class InsertionSort

**Comment by cegerard** [comment]
@Sylvain-Reig As you make another PR I simply close this one.

**Pull request closed without merging**